napari/layers/_layer_actions.py

```diff
@@ -37,7 +37,12 @@ def _duplicate_layer(ll: LayerList):
 
     for lay in list(ll.selection):
         new = deepcopy(lay)
```

Comment:
I wonder if this shouldn't be a method on layer?  this layer action isn't exactly the _only_ or "official" way to duplicate a layer.  You could also modify the layer's __deepcopy__ method if this is something that should always be applied?

Reply:
when overwrite `__deepcopy__` there is no option for `super().__deepcopy__` and I'm not sure how to write generic version of `__deepcopy___` for labels. 

I also think that the real copy method should use `_get_state` method, but I remember that there were some problems connected with this (interpolation in 3d etc).